make/langtools/netbeans/nb-javac/test/global/SwitchPrimitiveTypeTest.java

```diff
@@ -68,7 +68,7 @@ public void testSwitchOnPrimitiveType() throws IOException {
 
         String code = """
             class X {
-                public int checkType(int v) {
```

Comment:
The test is called `testSwitchOnPrimitiveType` - changing the parameter to `Number` makes it no longer a primitive type check.

Reply:
Test removed. The original test is no longer valid on JDK 19.